WordPress/Classes/ViewRelated/Views/BottomSheet/Demo/LocationResultsTableViewProvider.swift

```diff
@@ -0,0 +1,115 @@
+
+import UIKit
+
+// MARK: - LocationResultsMessages
+
+struct LocationResultsMessages {
```

Comment:
I love this. It removes a lot of noise. There are places in the codebase where this is done with an enumeration, instead of a struct, I think the rationale is because, being an enumeration, it can not be instantiated.